docs/DeveloperGuide.adoc

```diff
@@ -385,7 +385,55 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
-=== [Proposed]
+=== [Proposed] Calendar Viewer Feature
+==== Current Implementation
+The Calendar Viewer mechanism is facilitated by `Google Calendar API` and reside in the `ModelManager`.
+It supports viewing/add/editing/deleting capability that modifies the state of the NUSCouples App.
+Firstly, it uses Outh 2.0 endpoints to allow users to share specific data with the application while keeping their usernames, passwords, and other information private. For example, an application can use OAuth 2.0 to obtain permission from users to store files in their Google Drives which sync to the calendar.
+The only downside to this implementation is that the user needs to connect to the internet because NUSCouples app needs to open the system browser and supply a local redirect URI to handle responses from Google's authorization server.
```

Comment:
Maybe can put in design considerations (alternative w & w/o internet)? or don't call it a downside cause we supposed to +++ our product, instead mention in NFR that our app requires internet.

Reply:
I do not think it need to be mentioned actually. Since our user profile is university students who are proficient in CLI and typing, i would think it is a given that they have a laptop and internet, especially since NUS has free wifi.

Reply:
it needs to mentioned because im writing the description and behaviour of this Auth2.0

Reply:
i've removed the keyword downside already